templates/api/clients/node/src/client-helpers.ts.tpl

```diff
@@ -27,9 +27,9 @@ export interface ClientOptions {
 /**
  * @param accessToken The token to use to authenticate all requests the client submits
  * @param options The client options that affect its behavior
- * @returns The newly created {{ title .Config.Name }}Client instance
+ * @returns The newly created {{ stencil.ApplyTemplate "serviceNameLanguageSafe" }}Client instance
```

Comment:
Nit: you could avoid running this template everywhere by setting it to a variable.

e.g. add the below at. Close to the top of the file and reference it throughout.

```
{{- $serviceName := stencil.ApplyTemplate "serviceNameLanguageSafe" }}
```